<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myWizard</class>
 <widget class="QDialog" name="myWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/wizard/pic/res/wizardBackGroud.png);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_user">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pass">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>fuck you</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pass_confirm">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆留空</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>快速注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退        出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登       陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_show_info">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_show_gif">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_complete">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始体验</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_phone">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置手机</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_phone_brife">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>手机型号</string>
   </property>
   <property name="whatsThis">
    <string>这里输入手机型号</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_phone_detail">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>手机品牌</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_other_phone">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>没有找到我的手机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mail">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>xxx@xxx.com</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
